contracts/.solhintignore-test

```diff
@@ -0,0 +1,26 @@
+./src/v0.8/automation/
```

Comment:
For now it includes everything but CCIP. This does mean it runs by default for any new project